definitions/tools/pvacvector.cwl

```diff
@@ -84,6 +84,16 @@ inputs:
         inputBinding:
             prefix: "--n-threads"
         default: 8
+    spacers:
+        type: string?
+        inputBinding:
+            prefix: "--spacers"
+        default: "None,AAY,HHHH,GGS,GPGPG,HHAA,AAL,HH,HHC,HHH,HHHD,HHL,HHHC"
```

Comment:
Would it make sense to make this a `string[]` with `itemSeparator: ","` instead?